### PR TITLE
Fix install docs

### PR DESCRIPTION
**Pull request by pierallard**

### CODE REVIEW

File: install_pim/crontab_tasks_ee.rst.inc [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Configuring tasks via crontab
+-----------------------------
+
+The application needs the following tasks to be executed in background on a regular basis.
+
+Edit your crontab with ``crontab -e`` and configure your tasks.
+
+.. warning::
+
+    Be sure to update the cron of the same user you installed the PIM, to be able to run the command
```
Comment (rybus) [inline install_pim/crontab_tasks_ee.rst.inc]: of the user used to install the PIM

File: install_pim/crontab_tasks_ee.rst.inc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+You have add add these 3 commands to your local installation:
```
Comment (rybus) [inline install_pim/crontab_tasks_ee.rst.inc]: you have to add

File: install_pim/crontab_tasks_ee.rst.inc [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+In the following example, these lines will run the completeness, versioning and rules calculation every 15 minutes:
+
+.. code-block:: bash
+    :linenos:
+
+    # m  h  dom  mon  dow  command
+    */15 *  *    *    *    php /path/to/installation/pim-enterprise-standard/bin/console pim:completeness:calculate --env=prod > /path/to/installation/pim-enterprise-standard/app/logs/calculate_completeness.log 2>&1
+    */15 *  *    *    *    php /path/to/installation/pim-enterprise-standard/bin/console pim:versioning:refresh --env=prod > /path/to/installation/pim-enterprise-standard/app/logs/refresh_versioning.log 2>&1
+    */15 *  *    *    *    php /path/to/installation/pim-enterprise-standard/bin/console akeneo:rule:run --env=prod > /path/to/installation/pim-enterprise-standard/app/logs/rule_run.log 2>&1
```
Comment (rybus) [inline install_pim/crontab_tasks_ee.rst.inc]: Is it a good idea to execute all of them at the very same time?

Comment (rybus) [reply install_pim/crontab_tasks_ee.rst.inc]: okay never mind, your warning bellow says it all ^^

File: install_pim/installation_archive.rst.inc [hunk 4]
```diff
@@ -11,17 +11,18 @@ Extracting the archive
...
 
 .. note::
     The PIM will be extracted in the folder /path/to/installation/pim-community-standard.
 
 Initializing Akeneo
 -------------------
 
+.. note::
+
+    Remember that ``dev`` is the default environment. So when you launch a Symfony command, always add ``--env=prod`` in prod environment to avoid useless logging and profiling.
```
Comment (rybus) [inline install_pim/installation_archive.rst.inc]: and slowness

File: install_pim/installation_archive.rst.inc [hunk 5]
```diff
@@ -71,6 +70,8 @@ Known issues
 
  * not enough memory can cause the JS routing bundle to fail with a segmentation fault. Please check with ``php -i | grep memory`` that you have enough memory according to the requirements
 
+ * If you do an install the PIM twice, ensure all the PIM cron tasks are stopped before the installation of the database, to avoid `Base table or view not found` errors.
```
Comment (rybus) [inline install_pim/installation_archive.rst.inc]: If you do install the PIM twice | if you do an installation of the PIM twice

File: install_pim/installation_archive_ee.rst.inc [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+If an error occurs, it means that something went wrong in one of the previous steps. Please check error outputs of all the steps.
+
+Known issues
+------------
+
+ * with XDebug on, the default value of max_nesting_level (100) is too low and can make the ACL loading fail (which causes 403 HTTP response code on every application screen, even the login screen). A working value is 500: ``xdebug.max_nesting_level=500``
+
+ * not enough memory can cause the JS routing bundle to fail with a segmentation fault. Please check with ``php -i | grep memory`` that you have enough memory according to the requirements
+
+ * If you do an install the PIM twice, ensure all the PIM cron tasks are stopped before the installation of the database, to avoid `Base table or view not found` errors.
```
Comment (rybus) [inline install_pim/installation_archive_ee.rst.inc]: See previous command about the same sentence

File: install_pim/crontab_tasks.rst.inc [hunk 7]
```diff
@@ -1,34 +1,35 @@
...
-The application needs the following tasks to be executed in background on a regular basis:
+The application needs the following tasks to be executed in background on a regular basis.
+
+Edit your crontab with ``crontab -e`` and configure your tasks.
+
+.. warning::
+
+    Be sure to update the cron of the same user you installed the PIM, to be able to run the command
+
+You have add add these 2 commands to your local installation:
```
Comment (tamarasaurus) [inline install_pim/crontab_tasks.rst.inc]: says add twice

File: install_pim/installation_archive_ee.rst.inc [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+    $ yarn run webpack
+
+.. note::
+    The ``../composer.phar install`` command writes the database configuration to ``app/config/parameters.yml``
+
+.. include:: ./crontab_tasks_ee.rst.inc
+
+Testing your installation
+-------------------------
+Access the `web` folder using your favorite web browser and log in with *admin/admin*. If you see the dashboard, congratulations, you have successfully installed Akeneo PIM! You can also access the dev environment on using `app_dev.php` as index.
```
Comment (pandino101) [inline install_pim/installation_archive_ee.rst.inc]: "access the dev environment on using" remove the ON

File: install_pim/installation_archive_ee.rst.inc [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+**********************
+
+.. code-block:: bash
+    :linenos:
+
+    $ mkdir -p /path/to/installation
+    $ tar -xvzf pim-enterprise-standard-v2.0.tar.gz -C /path/to/installation/
+
+.. note::
+    Replace */path/to/installation* by the path to the directory where you want to install the PIM.
```
Comment (pandino101) [inline install_pim/installation_archive_ee.rst.inc]: "Replace with" instead of "Replace by"

File: install_pim/installation_archive_ee.rst.inc [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+
+.. note::
+    Replace *pim-enterprise-standard-v2.0.tar.gz* by the location and the name of the archive you have downloaded from `Partner portal <https://partners.akeneo.com/login>`_
```
Comment (pandino101) [inline install_pim/installation_archive_ee.rst.inc]: "Replace with" instead of "Replace by"

**Pull request merged**